src/main/java/com/globant/carrito/security/SecurityService.java

```diff
@@ -14,7 +21,30 @@
 	@RequestMapping(value = "/service/login", method=RequestMethod.POST)
 	@ResponseBody
 	public StatusDto login(@RequestBody LoginDto dto) {
-		return new StatusDto(dto.getUsername().equals("bart")
-				&& dto.getPassword().equals("simpson"));
+		
+		
+		EntityManagerFactory emf = Persistence
+				.createEntityManagerFactory("db");
+
+		EntityManager em = emf.createEntityManager();
+		
+		String username = dto.getUsername();
+		
+		TypedQuery<Clients> query = em.createQuery("from Clients c where c.username : username", Clients.class);
+		query.setParameter("username", username);
+		
+		if(query.getSingleResult() != null) {
+			Clients client = query.getSingleResult();
+			em.close();
+			return new StatusDto(dto.getUsername().equals(client.getUserName())
+					&& dto.getPassword().equals(client.getPassword()));
+		}else{
+			return new StatusDto(false);
+		}
+	}
+	
+	@RequestMapping(value = "/service/logout", method=RequestMethod.POST)
```

Comment:
El logout podría ser GET, porque no lleva ningún mensaje... pero si querés dejarlo en POST por "sentido de simetría", está bien también.

http://seekcartoon.com/watch/26767-the-simpsons-season-12-episode-18-trilogy-of-error.html#.VNIuGOpGjUY (minuto 5:35) jajaja